src/main/java/seedu/progresschecker/model/ProgressChecker.java

```diff
@@ -72,12 +72,12 @@ public void resetData(ReadOnlyAddressBook newData) {
         try {
             setPersons(syncedPersonList);
         } catch (DuplicatePersonException e) {
-            throw new AssertionError("AddressBooks should not have duplicate persons");
+            throw new AssertionError("ProgressCheckers should not have duplicate persons");
```

Comment:
The change followed the convention of having the name as plural but does it make more sense for the name to be singular in this case?

Reply:
yes